test/dataloaders/data4.txt.exe

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
this fails on my machine: `env: python: No such file or directory`

(maybe I should fix my machine, though)

Reply:
I did this to get the test to pass in CI. I don't really want to get into the business of customizing the Window's CI images if we can avoid it.

Reply:
I'm not sure this is the right call, and in any case if we change from python3 to python, we must also change the documentation that currently says:

> If you use `.py`, `.R`, `.rs`, or `.go`, the corresponding interpreter (`python3`, `Rscript`, `rust-script`, or `go run`, respectively) must be installed and available on your `$PATH`.

Reply:
related: #935

Reply:
I don't think we need to adjust the docs. This isn't a python data loader, and those will continue to work as is. This is just an arbitrary exe data loader that we use in our tests, I didn't modify the Python support.

Reply:
fixed by using `perl` which is available on both CI images and on macos for local devs.